cognite/client/_api/datapoints.py

```diff
@@ -625,6 +625,7 @@ def insert_dataframe(self, dataframe, external_id_headers: bool = False):
             dataframe (pandas.DataFrame):  Pandas DataFrame Object containing the time series.
             external_id_headers (bool): Set to True if the column headers are external ids rather than internal ids.
                 Defaults to False.
+            dropna (bool): Set to True to skip NaN and inf in DataFrame and post valid data
```

Comment:
I don't think `dropna` should remove +/- inf, - it is unexpected behavior (btw pandas has an option called `use_inf_as_na`).
I also don't have a perfect solution to this, but a couple of suggestions:
- Have two separate args: `dropna` and `dropinf`
- Have one arg for both cases: `drop_invalid`
I like having `dropna` available because it is sooo well known in the python/pandas community. However, I also like only having a single arg for this... hmm 🤔 

(Let's decide on this before doing a PR review!)

Reply:
I agree, a third approach would be either let infs throw an error. In my experience they don't often show up in everyday DS work. Although it seems ugly to have this option for NaN but not inf....

Reply:
I think I like your third option the best; keep current logic where an error is raised on infinities, and just implement `dropna`.

Reply:
Have updated with that